src/main/resources/META-INF/rewrite/hamcrest.yml

```diff
@@ -87,6 +87,23 @@ recipeList:
       matcher: isA
       assertion: isInstanceOf
 
+  # Array Matchers
+  - org.openrewrite.java.testing.hamcrest.HamcrestMatcherToAssertJ:
+      matcher: arrayContaining
+      assertion: containsExactly
+  - org.openrewrite.java.testing.hamcrest.HamcrestMatcherToAssertJ:
+      matcher: arrayContainingInAnyOrder
+      assertion: containsOnly
```

Comment:
As discussed:
> Hamcrest arrayContainingInAnyOrder
Creates an order agnostic matcher for arrays that matches when each item in the examined array is logically equal to one item anywhere in the specified items. For a positive match, the examined array must be of the same length as the number of specified items.

versus

> AssertJ containsOnly
Verifies that the actual group contains only the given values and nothing else, in any order and ignoring duplicates (i.e. once a value is found, its duplicates are also considered found).
> If you need to check exactly the elements and their duplicates use:
containsExactly(Object...) if the order does matter
containsExactlyInAnyOrder(Object...) if the order does not matter

which I'm not 100% sure, but sounds like maybe it should be contansExactlyInAnyOrder? or what do you think there?

We might want to similarly check `hasItemInArray` versus `contains` below for any semantic differences.